src/features/areaAssignments/components/OrganizerMapRenderer.tsx

```diff
@@ -398,7 +383,11 @@ const OrganizerMapRenderer: FC<OrganizerMapRendererProps> = ({
             return (
               <>
                 {overlayStyle == 'households' && (
-                  <DivIconMarker iconAnchor={[0, 0]} position={mid}>
+                  <DivIconMarker
+                    iconAnchor={[0, 0]}
+                    position={mid}
+                    zIndexOffset={100}
```

Comment:
What do you think about setting this on all `DivIconMarker`, or exposing this prop but having a default value, so it's always higher than other layers but it can be overridden.

Reply:
(we talked in person and decided against this). But we did make the props for `DivIconMarker` nicer.